### PR TITLE
sysutils/tt: New port

### PR DESCRIPTION
**Pull request by 0x501D**
Command-line utility to manage Tarantool applications.

### CODE REVIEW

**Comment by arrowd** [comment]
@wahjava You're knowledgeable in Go, would mind looking at this?

File: sysutils/tt/Makefile [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+BUILD_DEPENDS=  ${LOCALBASE}/bin/mage:devel/mage
+
+USES=		go:modules
+
+PLIST_FILES=	bin/tt
+
+do-build:
+	@cd ${WRKSRC} && ${SETENV} ${MAKE_ENV} ${GO_ENV} ${LOCALBASE}/bin/mage build
+
+GH_TUPLE=	\
```
Comment (metalefty) [inline sysutils/tt/Makefile:27]: Now we can use GO_MODULE instead of GH_TUPLE.

Comment (wahjava) [reply sysutils/tt/Makefile:27]: Yep, it'll also help with fetching modules which aren't on GitHub.

**Review by wahjava — CHANGES_REQUESTED** [review]

File: sysutils/tt/Makefile (hunk 1 above)
Comment (0x501D) [reply sysutils/tt/Makefile:27]: Unfortunately, tt does not work as a go module due to the complexity and structure of the project itself:
```sh
$ go get github.com/tarantool/tt@latest
go: github.com/tarantool/cartridge-cli@v0.0.0-20220605082730-53e6a5be9a61 (replaced by ./cli/cartridge/third_party/cartridge-cli): reading cli/cartridge/third_party/cartridge-cli/go.mod: open /home/void/tmp/cli/cartridge/third_party/cartridge-cli/go.mod: no such file or directory
```

Comment (0x501D) [reply sysutils/tt/Makefile:27]: Is it possible to just remove `USES= go:modules`, GH_TUPLE and just do the mage build in the Makefile?

Comment (0x501D) [reply sysutils/tt/Makefile:27]: And if port absolutely need to build without a network, I can try to use a cache of vendor libraries, the `tt` repository has them for `tt` gentoo ebuild.

**Pull request closed without merging**